requirements.txt

```diff
@@ -20,3 +20,5 @@ jmespath==0.10.0
 textract
 html5lib==1.1
 click==8.0.2
+pyarrow==8.0.0
```

Comment:
After reinitializing the conda environment, some flows failed because pandas needs `fastparquet` or `pyarrow` for parquet support.  I actually tried `fastparquet` first and it seemed to work, except the UpdateParquetFiles task generated corrupt parquet files. 😬 Switching to `pyarrow` seems to work though.